### PR TITLE
E-990 fix: remove GitHub auth method

### PR DESCRIPTION
**Pull request by ifiri**

### CODE REVIEW

**Comment by ifiri** [comment]
@dudiq ready for review

File: package.json [hunk 1]
```diff
@@ -68,7 +68,6 @@
     "bignumber.js": "^9.0.0",
```
Comment (dudiq) [inline package.json:68]: what about `VUE_APP_GIT_CLIENT_ID`?

Comment (ifiri) [reply package.json:68]: missed

Comment (lamartire) [reply package.json:68]: @ifiri remove it from each `.env` file

**Comment by ifiri** [comment]
@dudiq @lamartire done

**Pull request merged**